src/window.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+** EPITECH PROJECT, 2022
+** indieStudio [WSL : Ubuntu]
+** File description:
+** window
+*/
+#include "Window.hpp"
+
+
+indie::Window::Window(int screenWidth, int screenHeight, char *name) {
```

Comment:
Use constant reference of std::string instead of char *

Reply:
yes, but InitWindow require tu put a const char* , i'm trying to correct it but i have some issues with my mind x). i will find a way somehow

Reply:
You can just send name.c_str()